srcs/plugin.video.playermb/main.py

```diff
@@ -109,6 +110,7 @@ def load_ints(path):
 SUBTITLEFILE = os.path.join(DATAPATH, 'temp.sub')
 M3UFILE = addon.getSetting('m3u_fname')
 M3UPATH = addon.getSetting('m3u_path')
+M3UOVERWRITE = addon.getSetting('m3u_overwrite')
```

Comment:
Nie wiem czy to a duży sens, ale mamy funkcję `get_bool()`, która jest o tyle niepotrzebne, bo sam `xbmcaddon.Addon` ma [`getSettingBool()`](https://alwinesch.github.io/group__python__xbmcaddon.html#ga8b2b9a6b4b08f6e79c893e5255e0bf7a).

Wtedy nie trzeba by porównywać niżej z `'true'`.

Reply:
dzieki, poprawione